docs/feature-gates.md

```diff
@@ -209,59 +209,22 @@ there is a risk of conflicts in CIDR allocation between the two.
 
 ### AntreaIPAM
 
-`AntreaIPAM` is a feature that allows flexible control over Pod IP Addressing. This can
-be achieved by configuring custom resource `IPPool` with desired set of IP ranges.
-The pool can be annotated to Namespace, and Antrea will manage IP address assignment for
-corresponding Pods according to pool spec. In the future, support will be extended to
-Deployments and StatefulSets, as well as requesting specific IP via Pod annotation.
+`AntreaIPAM` feature allows flexible control over Pod IP addressing. This can be
+achieved by configuring `IPPool` CRD with a desired set of IP ranges. The `IPPool` can be
+annotated to Namespace, Pod and PodTemplate of StatefulSet/Deployment. Antrea will manage
+IP address assignment for corresponding Pods according to `IPPool` spec.
 
-Note that IP pool annotation can not be updated, but rather re-created. IP Pool can be
-extended, but cannot be shrunk if already assigned to a resource. The IP ranges of IP
-Pools must not overlap, otherwise it would lead to undefined behavior.
+Note that the IP pool annotation cannot be updated or deleted without recreating the
+resource. An `IPPool` can be extended, but cannot be shrunk if already assigned to a
+resource. The IP ranges of IPPools must not overlap, otherwise it would lead to undefined
+behavior.
 
-Regular `Subnet per Node` IPAM will continue to be used for resources without IP pool
+Regular `Subnet per Node` IPAM will continue to be used for resources without the IP pool
```

Comment:
As they are duplicate with the doc in antrea-ipam.md, can we remove the 2nd and 3rd paragraphs here and add a link in the end of the first paragraph like other features? "Refer to this [document](url) for more information."

Reply:
Sure. Fixed.